db/migrate/20181025153846_devise_token_auth_create_users.rb

```diff
@@ -0,0 +1,59 @@
+class DeviseTokenAuthCreateUsers < ActiveRecord::Migration[5.2]
```

Comment:
This wasn't in the last pr too 🤔  ?

Reply:
nup, the devise_token_auth gem was install in this branch and when I executed the installation It replaced the user migration for this. So this is "new".

Reply:
wait, so you updated a migration file?